native-addons-are-close-to-being-on-par-with-js-modules.md

```diff
@@ -0,0 +1,113 @@
+# Native Addons Are Close To Being On Par With JS Modules
+
+Native addons have been a part of Node.js since the very beginning. So have JS modules 😉 So, what does it mean that they are now close to being "on-par"? Why only now?
```

Comment:
Please try to avoid using the word so.

Reply:
Consider following why how what. This seems to begin with what & how.

Reply:
I've now added a blurb right after the title that introduces the "why". I intentionally wish to set a conversational tone by using the word "So".